Formula/ec2-api-tools.rb

```diff
@@ -18,7 +18,7 @@ class Ec2ApiTools < Formula
   # Deprecated upstream somewhere between 2017-12-24 and 2018-09-09 here:
   # https://docs.aws.amazon.com/AWSEC2/latest/APIReference/Welcome.html
   # The current version (1.7.5.1) is from 2015-09-08.
-  deprecate! date: "2020-02-03", because: :deprecated_upstream
+  disable! date: "2022-07-31", because: :deprecated_upstream
```

Comment:
For some formulae like `ec2-api-tools`, dependents were deprecated afterward (e.g. `aws-cfn-tools`, `elb-tools`).

Main question is how we want to deal with removal flow:
- we could disable dependents in this PR.
- we could keep them deprecated, maybe try to disable them before 1 year, and then finally remove dependency tree either at min/max of dates.

Reply:
I'm going to go for disabling them now. It's been more than 3 months already, so it's not conflicting with the policy.